### ISSUE TITLE
Is there a way to change the height of the window in the emscripten version?

### ISSUE THREAD

**Issue by christislord12** [danoon2/Boxedwine#71]
Is there a way to change the height of the window in the emscripten version? I created an application and am trying to get it to run in the emscripten version. The application is running great, the only problem is that the window created is not tall enough for the application. Is there a way to adjust the height? I attached an image so you can see what I am working with. 
![boxedwinetesting](https://user-images.githubusercontent.com/53404933/209418692-78131180-443f-4c76-bccf-586231ad238d.jpg)

**Comment by kevodwyer** [comment]
Yes. You can use the url parameter resolution ie boxedwine.html?resolution=800x600
